src/olympia/accounts/views.py

```diff
@@ -217,6 +218,22 @@ def inner(self, request):
     return outer
 
 
+def add_api_token_to_response(response, user):
+    # Generate API token and add it to the json response.
+    payload = jwt_api_settings.JWT_PAYLOAD_HANDLER(user)
+    token = jwt_api_settings.JWT_ENCODE_HANDLER(payload)
+    if hasattr(response, 'data'):
+        response.data['token'] = token
+    # Also include the API token in a session cookie, so that it is available
+    # for universal frontend apps.
+    response.set_cookie(
+        'jwt_api_auth_token', token,
+        max_age=None, secure=settings.SESSION_COOKIE_SECURE or None,
+        httponly=settings.SESSION_COOKIE_HTTPONLY or None)
+
+    return response
+
+
 class LoginView(APIView):
 
     @waffle_switch('fxa-auth')
```

Comment:
The waffle will be removed by @mstriemer this quarter when we remove the FxA "migration" path and FxA login will be the only way to login to AMO.